src/shared/generator.c

```diff
@@ -538,6 +538,7 @@ int generator_hook_up_growfs(
                 "BindsTo=%%i.mount\n"
                 "Conflicts=shutdown.target\n"
                 "After=%%i.mount\n"
+                "After=systemd-repart.service"
```

Comment:
This cannot work, lacks `\n` at the end of the line.

That said please merge with previous line. i.e.

```
"After=systemd-repart.service %%i.mount\n"
```